master/Makefile

```diff
@@ -98,6 +100,7 @@ build: gen
 		./cmd/determined-master
 
 .PHONY: build-race
+build-race: build-proto
```

Comment:
I don't know if this syntax is correct or not, but I think it definitely makes more sense to put the target dependency on the last line, with the `gen` dependency.

```
build-race: export WHATEVER=
build-race: export WHOCARES=
build-race gen build-proto     # dependent targets go on this line
```